desktop/src/main.rs

```diff
@@ -483,13 +484,6 @@ impl App {
                                     window.request_redraw();
                                 }
                             }
-                            WindowEvent::CursorLeft { .. } => {
-                                let mut player_lock = player.lock().unwrap();
-                                player_lock.handle_event(PlayerEvent::MouseLeft);
-                                if player_lock.needs_render() {
-                                    window.request_redraw();
-                                }
-                            }
```

Comment:
Why remove this event? (Maybe the name `MouseLeft` is confusing with the `KeyCode::MouseLeft` -- we could rename this even to `MouseLeave` instead).

Reply:
For reference, this will be used at some point because AS3 has an [`Event.MOUSE_LEAVE`](https://help.adobe.com/en_US/FlashPlatform/reference/actionscript/3/flash/events/Event.html#MOUSE_LEAVE).